src/main/java/frc/robot/subsystems/arm/commands/SetElbowAngle.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.subsystems.arm.commands;
```

Comment:
Will this and the `SetShoulderAngle` commands be deleted after testing is done?

Reply:
Probably

Reply:
By the way, you're missing the `arm` object in the `addRequirements` method.